src/bodyElement.ts

```diff
@@ -136,6 +145,8 @@ function toSerialisable(elem: BodyElement): JsonSerialisable {
             return { ...elem };
         case ElementKind.Embed:
             return { ...elem, alt: optionToSerialisable(elem.alt) };
+        case ElementKind.LiveEvent:
+                return { ...elem };
```

Comment:
```suggestion
            return { ...elem };
```

Reply:
Also, if it doesn't cause you a type error, you may be able to leave this to the `default` case.